migration-from-4x.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: default
+title: Migrating from TinyMCE 4.x to TinyMCE 5.0.
+title_nav: Migrating from 4.x
+description: Instructions for migrating from TinyMCE 4.x to TinyMCE 5.0.
+keywords: migration considerations premigration pre-migration
+---
+
+## Migrating from 4.x
+
+The new TinyMCE 5.0 editor comes with significant changes to the previous versions, though most configuration changes affect complex use cases like adding custom components. Setting up a basic TinyMCE 5.0 instance should be no harder than with TinyMCE 4.x.
+
+Our team at Tiny has worked on creating a more configurable, more modern and less cumbersome editor while retaining the familiarity of the user interface from the older versions.
+
+This chapter describes the migration process and workarounds if you are using an older version of TinyMCE. It includes tasks that you must perform before the migration can begin, and different workaround procedures for deprecated, deleted, and updated features.
+
+## Editor-Core
+
+### Initialization
+
+The initialization process of TinyMCE 5.0 is the same as TinyMCE 4.x. The bootstrap process and initialization events all remain the same.  The main differences are in the `init` configuration, specifically the configuration items for UI components. It still retains a familiar JSON structure. However, many of the configuration options have been simplified.
```

Comment:
- "initialisation process" should really be quantified - what does this refer to? Adding the TinyMCE 5.0 script to the page? Setting up a basic config? 
- What do we mean by "bootstrap process and initialisation events"? Are these terms from somewhere else in the docs, in which case can we link to it for clarity?

Reply:
Done. Added a link to the Quickstart page.